docs/Operators.md

```diff
@@ -11996,13 +11998,13 @@ node = onnx.helper.make_node(
 
 x = np.array([1, 2, 3]).astype(np.float32)
 y = np.array([4, 5, 6]).astype(np.float32)
-z = np.power(x, y)  # expected output [1., 32., 729.]
+z = pow(x, y)  # expected output [1., 32., 729.]
```

Comment:
Please add tests for newly supported types.

Reply:
there are 2 tests in export_types() involving int64.
Would adding 2 others for int32 be sufficient?

Reply:
Your new tests are good enough. Thanks.